apps/logstream/src/main/java/com/datatorrent/apps/logstream/Application.java

```diff
@@ -158,7 +157,7 @@ private SYSTEM_KEYS(String value)
   {
     String daemonAddress = dag.getValue(DAG.GATEWAY_CONNECT_ADDRESS);
     if (!StringUtils.isEmpty(daemonAddress)) {
-      URI uri = URI.create("ws://" + daemonAddress + "/pubsub");
+      URI uri = URI.create((dag.getValue(DAG.GATEWAY_USE_SSL) ? "wss://" : "ws://") + daemonAddress + "/pubsub");
```

Comment:
Since this is changed in so many places, please add a utility method to centralize (regardless of core).

Reply:
@sandeshh @tweise We are taking a different tact in core so the method will not be there. Will add one here.